registry/quilt_server/views.py

```diff
@@ -427,6 +485,21 @@ def _get_or_create_customer():
 def _get_customer_plan(customer):
     return PaymentPlan(customer.subscriptions.data[0].plan.id)
 
+def team_route(rule, defaults={}, **options):
+    """
+    Decorator that adds the route itself and the corresponding legacy non-team route.
+    """
```

Comment:
This method is a bit tricky and could use a more detailed explanation in the comment. It wasn't obvious to me why two routes are needed (to support older clients) nor what the different URL patterns look like. Also, I think I get why, but it's worth a quick note on why you create and return the decorator function.